src/Storage/Storage.Management/ChangeLog.md

```diff
@@ -18,6 +18,20 @@
         - Additional information about change #1
 -->
 ## Upcoming Release
+* Support traling dot in Azure file and directory name by default
```

Comment:
Use "Supported trailing dot in Azure file and directory name by default"

Reply:
Updated.